CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to PaperQA
+
+Thank you for your interest in contributing to PaperQA!
+Here are some guidelines to help you get started.
+
+## Setting up the development environment
+
+We use [`uv`](https://github.com/astral-sh/uv) for our local development.
+
+1. Install `uv` by following the instructions on the [uv website](https://astral.sh/uv/).
+2. Run the following command to install all dependencies and set up the development environment:
+
+   ```bash
+   uv sync
+   ```
+
+## Installing the package for development
+
+If you prefer to use `pip` for installing the package in development mode, you can do so by running:
+
+```bash
+pip install -e .
+```
+
+## Running tests and other tooling
+
+Use the following commands:
+
+- Run tests:
```

Comment:
we should note that you can't run tests w.o at least an open ai key

Reply:
Nice suggestion, implemented this just now